runtime/onert/backend/CMakeLists.txt

```diff
@@ -5,10 +5,14 @@ add_subdirectory(cl_common)
 add_subdirectory(acl_common)
 
 # Backends
+set(LIB_ONERT_BACKEND_CPU onert_backend_cpu)
 add_subdirectory(cpu)
 add_subdirectory(acl_cl)
 add_subdirectory(acl_neon)
 add_subdirectory(ruy)
 add_subdirectory(gpu_cl)
 add_subdirectory(xnnpack)
 add_subdirectory(trix)
+
+# Backend to train
+add_subdirectory(train)
```

Comment:
train? training?

Reply:
I don't know which one have determined :(
If you want to change it to training, I will.